blueocean-dashboard/src/main/js/OrganizationPipelines.jsx

```diff
@@ -12,37 +12,34 @@ import * as pushEventUtil from './util/push-event-util';
 const { object, array, func, node, string } = PropTypes;
 
 class OrganizationPipelines extends Component {
-
-    // FIXME: IMO the following should be dropped
+    // FIXME: get rid of context use
     getChildContext() {
-        const {
-            params,
-            pipelines,
-        } = this.props;
-
-
-        // The specific pipeline we may be focused on
-        const pipeline = this._selectPipeline(pipelines, params);
-
+        if (this._getOrganizationName()) {
+            return {
+                pipelines: this.props.organizationPipelines,
+            };
+        }
         return {
-            pipelines,
-            pipeline,
+            pipelines: this.props.allPipelines,
         };
     }
-
+    
     componentWillMount() {
         const config = this.context.config;
         if (config) {
-            const { organization } = this.context.params;
-            this.props.fetchPipelinesIfNeeded(this.context.config, organization);
-
-            const _this = this;
-
+            const organizationName = this._getOrganizationName();
+            
+            if (organizationName) {
```

Comment:
@michaelneale not really sure, could be fine. seems like the main dashboard should just be a search or most recent activity, which could make it a different purpose than simply getting all the pipelines for an org